roles/ceph-fetch-keys/tasks/main.yml

```diff
@@ -11,10 +11,10 @@
   delegate_to: localhost
   become: false
 
-- name: "copy ceph user and bootstrap keys to the ansible server in {{ fetch_directory }}/{{ fsid }}/"
+- name: "copy ceph user and bootstrap keys to the ansible server in {{ fetch_directory | default('fetch/') }}/{{ fsid }}/"
   fetch:
     src: "{{ item }}"
-    dest: "{{ fetch_directory }}/{{ fsid }}/{{ item }}"
+    dest: "{{ fetch_directory | default('fetch/') }}/{{ fsid }}/{{ item }}"
```

Comment:
it's probably better to add back a `fetch_directory` variable in `./roles/ceph-fetch-keys/defaults/main.yml`

Reply:
Ok I'll do that. Wasn't sure what solution you preferred. Will take another day though.